tests/test_wrap.py

```diff
@@ -103,3 +104,62 @@ def test_cachedir(self):
         with self.assertRaises(errors.ResourceNotFound):
             fs.getinfo('/foofoo')
 
+    def test_cachedir_time(self):
```

Comment:
Are you familiar with https://github.com/spulec/freezegun ? You can get rid of those sleep in tests.